egs2/binaural_librispeech/enh1/local/data.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Aalborg University (Authors: Holger Severin Bovbjerg)
+# Apache 2.0
+set -e
+set -u
+set -o pipefail
+
+log() {
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+help_message=$(cat << EOF
+Usage: $0 [--min_or_max <min/max>] [--sample_rate <8k/16k>]
+  optional argument:
+    [--min_or_max]: min (Default), max
+    [--sample_rate]: 8k (Default), 16k
+EOF
+)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+cdir=$PWD
+
+# Path to the directory containing MUSAN noise
+musan=${MUSAN}
+
+# Path to the directory containing Binaural LibriSpeech
+binaural_librispeech="${BINAURAL_LIBRISPEECH}"
```

Comment:
Better to add a variable `BINAURAL_LIBRISPEECH` in [egs2/TEMPLATE/asr1/db.sh](https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/db.sh#L65) and add a value check like [this](https://github.com/espnet/espnet/blob/master/egs2/aishell4/enh1/local/data.sh#L81-L85).

Reply:
If the write permission of this directory is needed, specify the default value to `downloads`.